addons/account/models/account_cash_rounding.py

```diff
@@ -37,7 +37,7 @@ class AccountCashRounding(models.Model):
     rounding_method = fields.Selection(string='Rounding Method', required=True,
         selection=[('UP', 'UP'), ('DOWN', 'DOWN'), ('HALF-UP', 'HALF-UP')],
         default='HALF-UP', help='The tie-breaking rule used for float rounding operations')
-    company_id = fields.Many2one('res.company', related='profit_account_id.company_id')
+    company_id = fields.Many2one('res.company', required=True, default=lambda self: self.env.company)
```

Comment:
The company is not required (and shouldn't be), the fields that depend on the company are already `company_dependent=True`